S21/osamaibraheem11/s21.py

```diff
@@ -0,0 +1,26 @@
+import csv
+
+vowels = ["A","E","I","O","U","Á","É","Í","Ó","Ú"]
+
+def LowerCase(word,language): 
+  if ('ga' in language) and (word.startswith('n') or word.startswith('t')) and word[1:2] in vowels:
+    return word[:1].lower()+"-"+word[1:].lower()
+  if 'en' in language:
+    return word.lower()
+  if ('tr' or "az" in language):
+    return word.replace('I',"ı").lower()
+  if ('el' in language):
+    if word.endswith('Σ'):
+      return word.replace('Σ',"ς").lower()
+    else:
+      return word.replace('Σ',"σ").lower()
+  else:
+    word.lower()
+read_tsv = csv.reader(open("tests.tsv"), delimiter="\t") 
```

Comment:
Thanks for providing test code!  I'd recommend expanding the test cases, especially with tests that will catch the issues I raised above, and then re-running once those are fixed.